Podfile.lock

```diff
@@ -0,0 +1,37 @@
+PODS:
```

Comment:
Do we actually want to include the `Podfile.lock`? I assume not.

Reply:
Done.